desktop/dbgl-legacy/dbgl-legacy.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="dbgl-legacy"
 VERSION="0.83"
-HOMEPAGE="http://members.quicknet.nl/blankendaalr/dbgl/"
-DOWNLOAD="https://slackware.uk/~urchlay/src/dbgl083_generic.tar.gz"
+HOMEPAGE="https://dbgl.org/"
+DOWNLOAD="https://dbgl.org/download/src083.zip"
```

Comment:
I get an MD5SUM error here?

Reply:
Yes, Don't how how it slipped past me, but here is a fixed PR.